server/controllers/v1/tests/comment.test.ts

```diff
@@ -0,0 +1,160 @@
+import db from '../../../models';
+import {
+  dbSync,
+  makeUser,
+  makeValidComment,
+  makeValidPost,
+  makeValidUser,
+} from '../../../models/tests/testHelpers';
+import CommentController from '../comment';
+
+const commentController = new CommentController(db.Comment);
+
+beforeEach(async () => {
+  await dbSync().catch((err) => fail(err));
+});
+
+describe('Test v1 - Comment Controller', () => {
+  describe('test comment fetching', () => {
+    it('should return a list of comments', async () => {
+      const author = await makeValidUser();
+      const post = await makeValidPost(author.id);
+      const comments = [
+        await makeValidComment(author.id, post.id),
+        await makeValidComment(author.id, post.id),
+        await makeValidComment(author.id, post.id),
+      ];
+      const result = await commentController.getComments(post.id, 100, 0);
+
+      expect(result.status).toBe(200);
+      for (let index = 0; index < 3; index++) {
+        expect(result.data.result![index].id).toBe(comments[index].id);
+        expect(result.data.result![index].User.firstName).toBe(
+          author.firstName
+        );
+      }
+      expect(result.data.total).toBe(3);
+    });
+
+    it('should return an error for a missing comment', async () => {
+      const result = await commentController.getComment('bingBong');
```

Comment:
```suggestion
      const result = await commentController.getComment('123-123');
```

This follows the structure of a UID making it more obvious what it is